web/css/json-editor.css

```diff
@@ -1,435 +1,448 @@
+       url("/resources/fixtures/fonts/overpass_light-webfont.woff") format("woff"),
+       url("/resources/fixtures/fonts/overpass_light-webfont.ttf") format("truetype"),
+       url("/resources/fixtures/fonts/overpass_light-webfont.svg#webfontzAU82Ltw") format("svg");
+  font-weight: 400;
+  font-style: normal;
 }
 body {
-    font-size: 16px;
-    font-family: "Overpass", Overpass, Helvetica, helvetica, arial, sans-serif;
-    font-weight: 100;
+  font-size: 16px;
+  font-family: "Overpass", Overpass, Helvetica, helvetica, arial, sans-serif;
+  font-weight: 100;
 }
 
 h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
-  font-family: "Overpass", Overpass, Helvetica, helvetica, arial, sans-serif;
-  color: #017EA3;
-  font-weight: 400;
+font-family: "Overpass", Overpass, Helvetica, helvetica, arial, sans-serif;
+color: #017EA3;
+font-weight: 400;
 }
 h1, .h1 {
-  color: #02546d;
+color: #02546d;
 }
 h1, .h1, h2, .h2 {
-  font-weight: 600;
+font-weight: 600;
 }
 a {
-  color: #017EA3;
+color: #017EA3;
 }
 a:hover, a:focus {
-  color: #0eb3e4;
+color: #0eb3e4;
 }
 
 
 .resize-overlay {
-    pointer-events: none;
-    position: relative;
+  pointer-events: none;
+  position: relative;
 }
 /* Styles for the navbar */
 .navbar-inverse .navbar-nav > li > a {
-    color: #fff;
-    font-weight: 400;
-    font-size: 16px;
+  color: #fff;
+  font-weight: 400;
+  font-size: 16px;
 }
 .navbar-inverse .navbar-header .navbar-brand:hover {
-    cursor: default;
-    color: #999;
+  cursor: default;
+  color: #999;
 }
 
 /* Styles for the pattern preview tool */
 #schema_holder .page-header {
-    padding-left: 60px;
+  padding-left: 60px;
 }
 #schema_holder + .alert {
-    margin: 20px 60px;
+  margin: 20px 60px;
 }
 #display_holder {
-  width: 100%;
-  border: 0;
+width: 100%;
+border: 0;
 }
 .sg-snippet-preview {
   position: relative;
   margin: 20px auto;
 }
 
 .sg-snippet-resize-handle {
-    height: 100px;
-    width: 30px;
-    margin-top: -50px;
-    position: absolute;
-    top: 50%;
-    -webkit-transition: all .4s;
-    -moz-transition: all .4s;
-    transition: all .4s;
+  height: 100px;
+  width: 30px;
+  margin-top: -50px;
+  position: absolute;
+  top: 50%;
+  -webkit-transition: all .4s;
+  -moz-transition: all .4s;
+  transition: all .4s;
 }
 
 .sg-snippet-resize-handle:before {
-    background: url('../img/icon-drag.svg') no-repeat;
-    position: absolute;
-    top: 50%;
-    content: '';
-    height: 17px;
-    margin-top: -8px;
-    width: 11px
+  background: url('../img/icon-drag.svg') no-repeat;
+  position: absolute;
+  top: 50%;
+  content: '';
+  height: 17px;
+  margin-top: -8px;
+  width: 11px
+}
+
+.sg-snippet-resize-handle.pos-right,
+.sg-snippet-resize-handle.pos-left {
+  background: rgba(192, 194, 196, 0.2);
+  height: 100%;
+  top: 0;
+  margin: 0;
+}
+
+.sg-snippet-resize-handle.pos-right::before,
+.sg-snippet-resize-handle.pos-left::before {
+  top: min(50%, 50vh);
```

Comment:
@starryeyez024 This is the magic right here.  It picks the minimum value of those 2 computations which gives us the nicest UI for the squishy handles

Reply:
+1 :)